src/render/dom/use-render.ts

```diff
@@ -7,7 +7,7 @@ import { buildSVGProps } from "./utils/build-svg-props"
 import { SVGVisualElement } from "./SVGVisualElement"
 import { isSVGComponent } from "./utils/is-svg-component"
 
-export function render<Props>(
+export function useRender<Props>(
```

Comment:
Why the rename to useRender? It's not really a hook, right?

Reply:
Technically the one we have defined for the DOM isn't a hook in that it doesn't call any React hooks. But I want to make it explicit that it does adhere to the rules of hooks and therefore they could be called from it, either for us and this DOM `useRender` or an externally-defined one.